### PR TITLE
Update sql-agent-extension-troubleshoot-known-issues.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by ujpat** [comment]
@bluefooted for review.

Customer has some PowerBI Images where SQL VM resource was registered in the past. It use to work in the past. We have blocked SQL VM Resource registration now (Confirmed with Yadi) for SKU images which are PowerBI/SSRS/SSAS. 

Customer is asking for a documentation. Added it to the current known issues doc.

**Comment by Court72** [comment]
@adbadram

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/sql-agent-extension-troubleshoot-known-issues.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ To repair the extension with the Azure portal:
 
    :::image type="content" source="media/sql-agent-extension-troubleshoot-known-issues/repair-extension.png" alt-text="Screenshot of the SQL IaaS Agent extension settings page of the SQL virtual machines extension in the Azure portal showing where to repair the extension.":::   
 
+## SQL Virtual Machine resource registration fails with error "Creating SQL Virtual Machine resource for PowerBI VM images is not supported"
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/sql-agent-extension-troubleshoot-known-issues.md]:
```suggestion
## SQL IaaS Agent extension registration fails with error "Creating SQL Virtual Machine resource for PowerBI VM images is not supported"
```

File: azure-sql/virtual-machines/windows/sql-agent-extension-troubleshoot-known-issues.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ To repair the extension with the Azure portal:
...
+
+Note that SQL VM Resource registration is blocked and not supported on PowerBI VM, SQL Server Reporting Server and SQL Server Analysis Service Images deployed from Azure Marketplace.
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/sql-agent-extension-troubleshoot-known-issues.md]:
```suggestion
Note that SQL IaaS Agent extension registration is blocked and not supported on PowerBI VM, SQL Server Reporting Server and SQL Server Analysis Service Images deployed from Azure Marketplace.
```

**Review by bluefooted — CHANGES_REQUESTED** [review]

**Comment by ujpat** [comment]
Thanks. I committed the changes

**Comment by ujpat** [comment]
#sign-off

**Comment by Court72** [comment]
Hi @ujpat - Thanks for your sign-off. In the public repo, pull requests should be approved by the author, another member of the content team, or a PM. 

@adbadram - Could you take a look? Thanks!

**Comment by bluefooted** [comment]
#sign-off

**Comment by bluefooted** [comment]
@Court72 Aditya is no longer on the team, I need to go through and change the author on all these articles. Can you please sign off for us?

**Pull request merged**